extension/test/e2e/pageObjects/MakePaymentFormPage.js

```diff
@@ -11,7 +11,7 @@ module.exports = class MakePaymentFormPage {
   async generateAdyenMakePaymentForm(clientKey) {
     await this.page.type('#adyen-client-key', clientKey)
     await this.page.$eval('#adyen-client-key', (e) => e.blur())
-    await this.page.waitForTimeout(10_000)
+    await this.page.waitForTimeout(2_000)
```

Comment:
Why this change? I guess there was a reason why we had 10 sec previously?

Reply:
I thought that is related to the review descr, now I think this is a different part 🤔 good question.
 
> Change from page.waitForTimeout(...) to frame.waitForSelector(...) to ensure Klarna payment page is completely loaded